a10_octavia/controller/worker/tasks/vthunder_tasks.py

```diff
@@ -140,7 +148,11 @@ class EnableInterface(VThunderBaseTask):
     @axapi_client_decorator
     def execute(self, vthunder):
         try:
-            self.axapi_client.system.action.setInterface(1)
+            interfaces = self.axapi_client.interface.get_list()
+            for i in range(len(interfaces['interface']['ethernet-list'])):
```

Comment:
Can we handle more exceptions? like dictionary key not found?

Reply:
Done

Reply:
We shouldn't be handling this KeyError. It's basically impossible to get without throwing the ACOSException error. If we are in the situation where this get's thrown, it's some catastrophic memory error that should never happen. We would want it to throw the raw exception which does still get logged.

Reply:
Removed the KeyError exception handling